backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/70-de.metas.fresh/5760680_sys_gh24287_AD_User_CreatedBy_IsSystemUser_Index.sql

```diff
@@ -0,0 +1,22 @@
+-- Run mode: SWING_CLIENT
+
+-- 2025-07-10T15:38:13.724Z
+INSERT INTO AD_Index_Table (AD_Client_ID,AD_Index_Table_ID,AD_Org_ID,AD_Table_ID,Created,CreatedBy,EntityType,IsActive,IsUnique,Name,Processing,Updated,UpdatedBy,WhereClause) VALUES (0,540819,0,114,TO_TIMESTAMP('2025-07-10 15:38:12.965000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y','N','AD_User_IsSystemUser','N',TO_TIMESTAMP('2025-07-10 15:38:12.965000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'IsSystemUser=''Y''')
+;
+
+-- 2025-07-10T15:38:13.797Z
+INSERT INTO AD_Index_Table_Trl (AD_Language,AD_Index_Table_ID, ErrorMsg, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy,IsActive) SELECT l.AD_Language, t.AD_Index_Table_ID, t.ErrorMsg, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy,'Y' FROM AD_Language l, AD_Index_Table t WHERE l.IsActive='Y'AND (l.IsSystemLanguage='Y' OR l.IsBaseLanguage='Y') AND t.AD_Index_Table_ID=540819 AND NOT EXISTS (SELECT 1 FROM AD_Index_Table_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Index_Table_ID=t.AD_Index_Table_ID)
+;
+
+-- 2025-07-10T15:39:02.479Z
+INSERT INTO AD_Index_Column (AD_Client_ID,AD_Column_ID,AD_Index_Column_ID,AD_Index_Table_ID,AD_Org_ID,Created,CreatedBy,EntityType,IsActive,SeqNo,Updated,UpdatedBy) VALUES (0,624,541454,540819,0,TO_TIMESTAMP('2025-07-10 15:39:01.747000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100,'D','Y',10,TO_TIMESTAMP('2025-07-10 15:39:01.747000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',100)
+;
+
+-- 2025-07-10T15:41:40.033Z
+UPDATE AD_Index_Table SET Name='AD_User_CreatedBy_IsSystemUser',Updated=TO_TIMESTAMP('2025-07-10 15:41:39.820000','YYYY-MM-DD HH24:MI:SS.US')::timestamp without time zone AT TIME ZONE 'UTC',UpdatedBy=100 WHERE AD_Index_Table_ID=540819
+;
+
+-- 2025-07-10T15:42:01.762Z
+CREATE INDEX AD_User_CreatedBy_IsSystemUser ON AD_User (CreatedBy) WHERE IsSystemUser='Y'
```

Comment:
i'm not sure i understand this index

Reply:
https://github.com/metasfresh/metasfresh/pull/21086